doc/api/fs.md

```diff
@@ -669,8 +669,10 @@ changes:
 * `callback` {Function}
   * `err` {Error}
 
-Asynchronous chmod(2). No arguments other than a possible exception are given
-to the completion callback.
+Asynchronously changes the mode of a file. No arguments other than a possible
```

Comment:
Maybe even go for “permissions” instead of “mode”? Is that clearer?

Reply:
Probably, yes.